compiler/rustc_mir_transform/src/check_unsafety.rs

```diff
@@ -574,7 +577,19 @@ pub fn check_unsafety(tcx: TyCtxt<'_>, def_id: LocalDefId) {
                     description,
                     unsafe_fn_msg,
                 );
-                err.span_label(source_info.span, description).note(note);
+
+                if let Some(help) = details.help_missing_feature(tcx) {
+                    if let Some(note_missing_features) = details.note_missing_feature(tcx) {
+                        err.span_label(source_info.span, description)
```

Comment:
You can move the `span_label` call above the `if let`, and move the `.help` between the two `if let`'s.

Reply:
Ok, I'm going to work on it.

Reply:
Done.